.vscode/launch.json

```diff
@@ -16,8 +16,9 @@
         "${workspaceRoot}/extensions/ql-vscode/out/**/*.js",
       ],
       "env": {
-        // uncomment to allow debugging the language server Java process from a remote java debugger
-        // "DEBUG_LANGUAGE_SERVER": "true"
+        // uncomment to debug the IDE or Query servers
+        // "IDE_SERVER_JAVA_DEBUG": "true",
```

Comment:
As a default, is it easier to comment these out or to set these to `false`?

Reply:
I can change them to false. Maybe makes more sense.